woodwork/column_accessor.py

```diff
@@ -246,6 +248,18 @@ def wrapper(*args, **kwargs):
                                                                                           invalid_schema_message,
                                                                                           'Series')
                         warnings.warn(warning_message, TypingInfoMismatchWarning)
+                elif _is_dataframe(result):
+                    # Initialize Woodwork if a valid table schema can be created from the original column schema
+                    col_schema = self.schema
+                    table_schema = TableSchema(column_names=[self.name],
+                                               logical_types={self.name: col_schema.logical_type},
+                                               semantic_tags={self.name: col_schema.semantic_tags},
+                                               column_metadata={self.name: col_schema.metadata},
+                                               use_standard_tags={self.name: col_schema.use_standard_tags},
+                                               column_descriptions={self.name: col_schema.description},
+                                               validate=False)
```

Comment:
Maybe another point towards implementing something in #965 that would let us initialize Woodwork directly here?

Reply:
Yep - added a comment to that issue related to this!